content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -0,0 +1,95 @@
+
+
+## The VolumeAttributesClass Resource
+
+Each VolumeAttributesClass contains the `driverName` and `parameters`, which are used when a PersistentVolume belonging to the class needs to be dynamically provisioned or modified.
+
+The name of a VolumeAttributesClass object is significant, and is how users can request a particular class. Administrators set the name and other parameters of a class when first creating VolumeAttributesClass objects. While the name of a VolumeAttributesClass object in a PersistentVolumeClaim is mutable, the parameters in an existing class are immutable.
+
+
+```yaml
+apiVersion: storage.k8s.io/v1alpha1
+kind: VolumeAttributesClass
+metadata:
+  name: silver
+driverName: pd.csi.storage.gke.io
+parameters:
+  provisioned-iops: "3000"
+  provisioned-throughput: "50" 
+```
+
+
+### Provisioner
+
+Each VolumeAttributes has a provisioner that determines what volume plugin is used for provisioning PVs. This field must be specified. 
+
+The feature support for VolumeAttributesClass is implemented in [kubernetes-csi/external-provisioner](https://github.com/kubernetes-csi/external-provisioner).
+
+You are not restricted to specifying the [kubernetes-csi/external-provisioner](https://github.com/kubernetes-csi/external-provisioner). You can also run and specify external provisioners, which are independent programs that follow a specification defined by Kubernetes. Authors of external provisioners have full discretion over where their code lives, how the provisioner is shipped, how it needs to be run, what volume plugin it uses (including Flex), etc. 
+
+
+### Resizer
+
+Each VolumeAttributes has a resizer that determines what volume plugin is used for modifying PVs. This field must be specified. 
+
+The modifying volume feature support for VolumeAttributesClass is implemented in [kubernetes-csi/external-resizer](https://github.com/kubernetes-csi/external-resizer).
+
+For example the existing PVC is using VolumeAttributesClass silver:
+
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: test-pv-claim
+spec:
+  …
+  volumeAttributesClassName: gold
+  …
+```
+
+
+A new VolumeAttributesClass gold is available in the cluster:
+
+
+```yaml
+apiVersion: storage.k8s.io/v1alpha1
+kind: VolumeAttributesClass
+metadata:
+  name: gold
+driverName: pd.csi.storage.gke.io
+parameters:
+  iops: "4000"
+  throughput: "60"
+```
+
+
+The end user can update the PVC with the new VolumeAttributesClass gold and apply:
+
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: test-pv-claim
+spec:
+  …
+  volumeAttributesClassName: gold
+  …
+```
+
+
+## Parameters
+
+Volume Classes have parameters that describe volumes belonging to the volume class. Different parameters may be accepted depending on the provisioner or the resizer. For example, the value `4000`, for the parameter `iops`, and the parameter `throughput` are specific to GCE PD. When a parameter is omitted, some default is used.
```

Comment:
> When a parameter is omitted, some default is used.

After a user apply the PVC with a different VolumeAttributesClass, if some parameters are omitted in the target VolumeAttributesClass, the default value of the parameters may be used but it is not guaranteed. Because its behavior is determined by a specific CSI driver implementation. Please refer to the related CSI driver documentation for more details.